code/src/java/pcgen/core/Equipment.java

```diff
@@ -5033,6 +5033,7 @@ else if (sizeDiff == 0)
 			}
 		}
 
+		assert wCat != null;
```

Comment:
Do we need this assert?

Reply:
I have decided to add it, because the code block down below fails, as it isn‘t prepared for NPEs. The null appears, when the code cannot find a wield category. I can replace it with a static check (if statement) that fails, when this happens and write into log how to fix it with LST.